tools/scripts/release/tagging.sh

```diff
@@ -37,4 +37,14 @@ for crate in ${updated_crates[@]}; do
 * [CHANGELOG](https://github.com/ockam-network/ockam/blob/${name}_v$version/implementations/rust/ockam/$name/CHANGELOG.md)";
 
     gh release create --draft --notes "$text" -t "$name v${version} (rust crate)" "$tag" --target $COMMIT_SHA
+}
+
+if [[ ! -z $TAG_SINGLE_CRATE ]]; then
+    echo "Tagging only $TAG_SINGLE_CRATE crate"
+    tag_crate $TAG_SINGLE_CRATE
```

Comment:
Hmm, I sort of wonder if tag / release should be different operations in the future, once we get to a "not every release is semver-breaking" situation.

That said, this is great for now.